### PR TITLE
feat(IPVC-2230): make changes to formats

### PR DESCRIPTION
**Pull request by sptaylor**
This PR: 
1. Removes the one-off `run_uta_...` scripts in bin 
2. Updates UTA GeneInfo format to store “gene_symbol” in addition to HGNC.
3. Renames “HGNC” in GeneAccessions to “gene_symbol”.
4. Updates related TxInfo format to store a GeneId  and rename hgnc to “gene symbol” that can be used to link records during database updates.
5. Makes sure the parsing scripts pull the correct fields for the new attributes

* Note - I did not change any of the impacted but archived scripts

### CODE REVIEW

File: loading/data/splign-manual/generate-loading-data [hunk 1]
```diff
@@ -79,6 +80,7 @@ def parse_splign(fn, txdata):
     txinfo = uta.formats.txinfo.TxInfo(
         origin=origin,
         ac=tx_ac,
+        gene_id=gene_id,
         hgnc=hgnc,
```
Comment (bsgiles73) [inline loading/data/splign-manual/generate-loading-data]: For `TxInfo` we renamed "hgnc" to "gene_symbol". Do we need to update this line?

File: loading/data/splign-manual/generate-loading-data [hunk 2]
```diff
@@ -64,6 +64,7 @@ def parse_splign(fn, txdata):
         raise KeyError(f"{tx_ac}: no cds or hgnc info in {txdata_fn}")
 
     hgnc = txd["hgnc"]
```
Comment (bsgiles73) [inline loading/data/splign-manual/generate-loading-data]: Perhaps we should rename "hgnc" to "gene_symbol" here to reduce confustion.

**Review by bsgiles73 — CHANGES_REQUESTED** [review]
Looks good, just a couple requests for changes. 

- Can we rename "hgnc" to "gene_symbol" in `sbin/ncbi-parse-gbff` and splign manual generate loading data scripts to reduce confusion.
- Update hgnc to gene symbol when initializing TxInfo.

File: loading/data/splign-manual/generate-loading-data (hunk 2 above)
Comment (sptaylor) [reply loading/data/splign-manual/generate-loading-data]: Done. I kept txd["hgnc"] as is since we haven't made changes the the txinfo.yaml file yet.

**Review by bsgiles73 — APPROVED** [review]
Looks good, thanks @sptaylor

**Pull request merged**